help/destinations/catalog/advertising/liveramp-distribution.md

```diff
@@ -110,24 +114,51 @@ To configure details for the destination, fill in the required and optional fiel
 >title="Client name"
 >abstract="Your advertiser account name, as you would like it to be shown to the destination partner. Use your company name. Do not use spaces or special characters."
 
-### [!DNL Amobee] {#amobee}
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the customer data fields for the Acast destination.](../../assets/catalog/advertising/liveramp-distribution/LR_Acast_DestSpecific.png)
+
+* **[!UICONTROL Client name]**: Your advertiser account name, as you would like it to be shown to the destination partner. Use your company name. Do not use spaces or special characters.
 
 ### [!DNL Ampersand.tv] {#ampersand-tv}
 
+>[!CONTEXTUALHELP]
+>id="company_name"
+>title="Your Company Name"
+>abstract="Your Company Name, as you would like it to be shown to the destination partner. Do not use spaces or special characters."
+
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the customer data fields for the Ampersand destination.](../../assets/catalog/advertising/liveramp-distribution/LR_Ampersand_DestSpecific.png)
+
+* **[!UICONTROL Your Company Name]**: Your Company Name, as you would like it to be shown to the destination partner. Do not use spaces or special characters.
```

Comment:
No need for camelcase in the description: "Your company name, as you would like it to be shown to the destination partner. Do not use spaces or special characters."